study.md

```diff
@@ -27,17 +27,18 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+ the HTML we write
+ the source of a webpage
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+yes because it's a visual representation of the DOM, it will look similar to HTML but it's not.
```

Comment:
It's a visual representation of the browser's implementation of the DOM, not the DOM itself.